src/Features/CSharp/Portable/MakeMethodSynchronous/CSharpMakeMethodSynchronousCodeFixProvider.cs

```diff
@@ -87,7 +87,7 @@ private static TypeSyntax FixMethodReturnType(IMethodSymbol methodSymbol, TypeSy
             return newReturnType;
         }
 
-        private static SyntaxTokenList FixMethodModifiers(SyntaxTokenList modifiers, ref TypeSyntax newReturnType)
+        internal static SyntaxTokenList FixMethodModifiers(SyntaxTokenList modifiers, ref TypeSyntax newReturnType)
```

Comment:
would prefer these move to a helper class that both providers can share.  in general we don't want providers calling into other providers.